gilbreth_perception/src/recognition_node.cpp

```diff
@@ -151,34 +152,48 @@ class RecognitionClass {
   }
 
   void loadCGConfig() {
+    //Compute Normals
     pcl::search::KdTree<PointType>::Ptr tree(new pcl::search::KdTree<PointType>());
     pcl::NormalEstimationOMP<PointType, NormalType> norm_est;
     std::vector<pcl::PointCloud<NormalType>::Ptr> model_normals_list;
 
     for (int i = 0; i < model_list.size(); i++) {
       pcl::PointCloud<NormalType>::Ptr model_normals(new pcl::PointCloud<NormalType>());
-      norm_est.setRadiusSearch(descr_rad);
+      norm_est.setKSearch(10);
```

Comment:
Either make this a configurable ROS parameter of make it into a static global constant.  I'd prefer the first option.

Reply:
Ok, will make it into the config YAML file.